### PR TITLE
Refactor saveUnitList() to not make a new ClientGUI

### PR DESCRIPTION
**Pull request by pavelbraginskiy**
Simple refactor of a previous pull request to not make the print-queue dialog depend on ClientGUI. 

Also sets the default filename of the new file to the name of the original file.

### CODE REVIEW

**Review by NickAragua — APPROVED** [review]

**Comment by HammerGS** [comment]
Conflicting files.

File: megameklab/src/megameklab/ui/dialog/PrintQueueDialog.java [hunk 1]
```diff
@@ -265,10 +270,34 @@ protected void okButtonActionPerformed(ActionEvent evt) {
...
+
+        File file = fileChooser.getSelectedFile();
+        if (!FilenameUtils.getExtension(file.getName()).equalsIgnoreCase(CG_FILEPATHMUL)) {
+            file = new File(file + "." + CG_FILEPATHMUL);
+        }
+
+        try {
+            EntityListFile.saveTo(file, entities);
+        } catch (IOException e) {
+            JOptionPane.showMessageDialog(getFrame(), "Error: " + e.getMessage());
```
Comment (rjhancock) [inline megameklab/src/megameklab/ui/dialog/PrintQueueDialog.java:299]: Would suggest using the MMLogger with (exception, Message, title) if this is going to be kept.

Also suggest using String.format() for the message.

Comment (rjhancock) [reply megameklab/src/megameklab/ui/dialog/PrintQueueDialog.java:299]: Honestly would suggest String.format() on any string concatenation.

Comment (pavelbraginskiy) [reply megameklab/src/megameklab/ui/dialog/PrintQueueDialog.java]: Done

**Pull request merged**